subsys/bluetooth/mesh/access.c

```diff
@@ -365,7 +365,23 @@ size_t bt_mesh_comp_page_0_size(void)
 	return size;
 }
 
-static int comp_add_elem(struct net_buf_simple *buf, struct bt_mesh_elem *elem,
+uint16_t bt_mesh_elem_addr_get(const struct bt_mesh_elem *elem)
+{
+	int offset;
+
+	offset = elem - dev_comp->elem;
+	if (offset > dev_comp->elem_count) {
```

Comment:
I believe Zephyr has a macro to test that an element belongs to a given array. That said, shouldn’t the test be `>=` since you’re dealing with a zero-based index? In fact, maybe it’s better to rename `offset` to `índex`?